components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -6692,7 +6692,7 @@ def argument_type(descriptorProvider, ty, optional=False, defaultValue=None, var
 
 def method_arguments(descriptorProvider, returnType, arguments, passJSBits=True, trailing=None, inCompartment=False):
     if needCx(returnType, arguments, passJSBits):
-        yield "cx", "*mut JSContext"
+        yield "cx", "JSContext"
```

Comment:
I suspect this might cause problems in the short term.

Reply:
Same as above. I thought we needed to change the type:

> - [ ] make the code generator declare trait methods that accept the new type instead of *mut JSContext (here and here; both the type )